src/Backend/Modules/MediaGalleries/Domain/MediaGallery/MediaGalleryType.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Domain\MediaGallery;
+
+use Backend\Core\Engine\Authentication;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Backend\Core\Engine\TemplateModifiers;
+use Backend\Form\Type\EditorType;
+use Backend\Modules\MediaGalleries\Domain\MediaGallery\Command\CreateMediaGallery;
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroupType;
+use Backend\Modules\MediaLibrary\ValueObject\MediaWidget;
+
+class MediaGalleryType extends AbstractType
+{
+    /** @var string */
+    private $dataClass;
+
+    /**
+     * MediaGalleryType constructor.
+     *
+     * @param string $dataClass
+     */
+    public function __construct(string $dataClass = CreateMediaGallery::class)
```

Comment:
this should be done with options not constructor parameters, for forms the constructor parameters should always be the same, if there are any, and it should be added in a service (I know content blocks doens't do that yet but that will be added in #2001